appdirs.py

```diff
@@ -13,8 +13,8 @@
 # - Mac OS X: http://developer.apple.com/documentation/MacOSX/Conceptual/BPFileSystem/index.html
 # - XDG spec for Un*x: http://standards.freedesktop.org/basedir-spec/basedir-spec-latest.html
 
-__version_info__ = (1, 4, 3)
-__version__ = '.'.join(map(str, __version_info__))
+__version__ = "1.4.3"
+__version_info__ = tuple(int(segment) for segment in __version__.split("."))
```

Comment:
Hi @mauritsvanrees!

Indeed the code is expecting integer version segments only.

I went thru all the history of this line and, `__version_info__` has always been [a triple of integers](https://github.com/leorochael/appdirs/commit/29a8f7d24b1334b3b789810dbf38808d866906b3).

In any case, whoever breaks the integer segments expectation will have a very early, reasonably obvious and clear error message in their hands, so I'm tempted to say YAGNI.

But I'm happy to defer to the maintainers on this.